src/sentry/api/endpoints/release_thresholds/release_threshold_index.py

```diff
@@ -0,0 +1,61 @@
+from django.db.models import Q
+from django.http import HttpResponse
+from rest_framework import serializers
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.organization import OrganizationEndpoint
+from sentry.api.paginator import OffsetPaginator
+from sentry.api.serializers import serialize
+from sentry.models.organization import Organization
+from sentry.models.release_threshold.release_threshold import ReleaseThreshold
+
+
+class ReleaseThresholdIndexGETValidator(serializers.Serializer):
+    environment = serializers.ListField(
+        required=False, allow_empty=True, child=serializers.CharField()
+    )
+    project = serializers.ListField(
+        required=True, allow_empty=False, child=serializers.IntegerField()
+    )
+
+
+@region_silo_endpoint
+class ReleaseThresholdIndexEndpoint(OrganizationEndpoint):
+    owner: ApiOwner = ApiOwner.ENTERPRISE
+    publish_status = {
+        "GET": ApiPublishStatus.PRIVATE,
```

Comment:
if we eventually want to publish this it should be EXPERIMENTAL

Reply:
awesome will update